### PR TITLE
Mobile_desktop_version

### PR DESCRIPTION
**Pull request by FarwaMuhibZada**
## Personal Portfolio

-  That is a personal Portfolio webpage. that have a main section that contaian all the articles 
-  navbar section which containe all the ancher tages and a logo picture. 
- have an about, projects, contact us section then 
- we use id selector to reference the specific section to specific links in navbar. 
- This web pages is fully responsive we use @media query that adjust our page in every screen size.
-  this is the main feature of this webpage and it has a greate design.

### CODE REVIEW

**Comment by sodabarauf** [comment]
Hi, @FarwaMuhibZada
I checked your portfolio, by my concept it is complete portfolio with beautiful design implemented, your hard work, dedication, and perseverance have brought you to where you are today.
this is truly amazing!

**Comment by FarwaMuhibZada** [comment]
> Hi, @FarwaMuhibZada I checked your portfolio, by my concept it is complete portfolio with beautiful design implemented, your hard work, dedication, and perseverance have brought you to where you are today. this is truly amazing!
@sodabarauf  thanks for your review and valuable feedback

**Comment by FarwaMuhibZada** [comment]
@ahzamir hello sir I hope you are doing well this is my portfolio website please check and kindly share your feedback thank you.

**Comment by ahzamir** [comment]
@FarwaMuhibZada 

- Although I said in the group that I will not provide feedback for you till you finish all of your project requirements:
  - Still, I want to give you some hints so while you are working on the mobile design you work on these sections too:
  - Your design is amazing:
    - Add background color while you scroll to your header search on how in Google.
    - Add mobile version design as it's not responsive for now.
    - Some of your buttons do not have functionality or links add them.
    - Most of your buttons and links are directed to index.html, add the exact link for all buttons and links LinkedIn should have your LinkedIn account link and the email should direct the user to the email.
    - Add your project screenshots and description in the projects section and include both source code (Github repo link) and live demo (deployed link).

**Comment by ahzamir** [comment]
@FarwaMuhibZada 

- I will check and provide complete feedback after you add the mobile version design and all remaining sections.

**Comment by FarwaMuhibZada** [comment]
> @FarwaMuhibZada
> 
> * Although I said in the group that I will not provide feedback for you till you finish all of your project requirements:
>   
>   * Still, I want to give you some hints so while you are working on the mobile design you work on these sections too:
>   * Your design is amazing:
>     
>     * Add background color while you scroll to your header search on how in Google.
>     * Add mobile version design as it's not responsive for now.
>     * Some of your buttons do not have functionality or links add them.
>     * Most of your buttons and links are directed to index.html, add the exact link for all buttons and links LinkedIn should have your LinkedIn account link and the email should direct the user to the email.
>     * Add your project screenshots and description in the projects section and include both source code (Github repo link) and live demo (deployed link).



> @FarwaMuhibZada
> 
> * I will check and provide complete feedback after you add the mobile version design and all remaining sections.

thank for your valuable feedback but I add all my project life demo on portfolio section when you Hover the mouse you can click there was a link .

**Comment by FarwaMuhibZada** [comment]
> > @FarwaMuhibZada
> > 
> > * Although I said in the group that I will not provide feedback for you till you finish all of your project requirements:
> >   
> >   * Still, I want to give you some hints so while you are working on the mobile design you work on these sections too:
> >   * Your design is amazing:
> >     
> >     * Add background color while you scroll to your header search on how in Google.
> >     * Add mobile version design as it's not responsive for now.
> >     * Some of your buttons do not have functionality or links add them.
> >     * Most of your buttons and links are directed to index.html, add the exact link for all buttons and links LinkedIn should have your LinkedIn account link and the email should direct the user to the email.
> >     * Add your project screenshots and description in the projects section and include both source code (Github repo link) and live demo (deployed link).
> 
> > @FarwaMuhibZada
> > 
> > * I will check and provide complete feedback after you add the mobile version design and all remaining sections.
> 
> @ahzamir  thank for your valuable feedback but I add all my project life demo on portfolio section when you Hover the mouse you can click there was a link .

**Comment by FarwaMuhibZada** [comment]
@ahzamir hello sir hope you are doing well  I create my website responsive for all screen size and  I also add my project source code link and  live demo project. but my link din account restricted for now  I don't know what is the problem so if the link din link in website doesn't open its because of that  I hope  I have done all the requirements thank you. 

Screenshot (761)

**Comment by FarwaMuhibZada** [comment]
@ahzamir  mention you for notification!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        I have completed two years of University, not only did I maintain excellent grades.
+      </p>
+        <br>
+    </div>
+  </section>
+
+  <!-- myProject sections -->
+  <section class="services" id="myProject">
+    <div class="main-text">
+      <h2><span>My</span>MyProject</h2>
```
Comment (ahzamir) [inline index.html]: - Solve the typo issue `My My project` Remove one ` My`

Screenshot 2024-06-06 210220

File: index.html [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            Check Out 10 best Design s update for the top web design and development companies
+          </p>
+          <a href="https://farwamuhibzada.github.io/Technical_documentation_page/">
+            <i class="fa-solid fa-arrow-up-right-from-square"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+
+  </section>
```
Comment (ahzamir) [inline index.html]: - Remove this section as it is not useful for now:

Screenshot 2024-06-06 210244

**Review by ahzamir — APPROVED** [review]
### Hi @FarwaMuhibZada 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- Check the comments below.

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by ahzamir** [comment]
@FarwaMuhibZada 

- Well done farwa, after you implement the feedback you can merge and deploy your project but don't forget to share your project on FB group and on your LinkedIn account.

**Comment by ahzamir** [comment]
> @ahzamir hello sir hope you are doing well I create my website responsive for all screen size and I also add my project source code link and live demo project. but my link din account restricted for now I don't know what is the problem so if the link din link in website doesn't open its because of that I hope I have done all the requirements thank you.
> 
> Screenshot (761)

- The problem is that you used someone else photo or designed illustration or vector under your profile maybe your profile picture so LinkedIn wants to make sure it's you.
  - I suggest you add your own profile picture.

**Comment by FarwaMuhibZada** [comment]
@ahzamir  thanks for your valuable feed back I implemented your feedback it helped me to make my website  better.

**Pull request merged**